constants/constants.go

```diff
@@ -87,3 +87,14 @@ var ActionTypeMap = map[ActionType]string{
 const (
 	InvalidEmailOrPassword = "Invalid email or password"
 )
+
+// <----------- constants for email --------------->
+
+// OTPEmailSubject ...
+const OTPEmailSubject = "Subject: One Time Password\n"
+
+// OTPEmailFrom ...
+const OTPEmailFrom = "From: iReflect<no-reply@ireflect.com>\n"
```

Comment:
This should be same for the complete iReflct application.
```suggestion
const EmailFrom = "From: iReflect<no-reply@ireflect.com>\n"
```

Reply:
done.